README.md

```diff
@@ -1,9 +1,6 @@
-
-# Swiboe
+# Swiboe ¬ [![Build Status](https://travis-ci.org/swiboe/swiboe.svg)](https://travis-ci.org/swiboe/swiboe) [![Join the chat at https://gitter.im/swiboe/swiboe](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/swiboe/swiboe?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
The ¬ is a bit weird. It seems like it is part of the name. If you want to separate the images from the name, I'd say use - or •